demisto_sdk/commands/common/errors.py

```diff
@@ -1203,6 +1205,18 @@ def integration_not_runnable():
         return "Could not find any runnable command in the integration." \
                "Must have at least one command, `isFetch: true`, `feed: true`, `longRunning: true`"
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_uuid(task_key, id, taskid):
+        return f"On task with the following key: {task_key}, the field of taskid: {taskid} and the feild of id: " \
+               f"{id} must be from uuid type."
+
+    @staticmethod
+    @error_code_decorator
+    def taskid_different_from_id(task_key, id, taskid):
+        return f"On task with the following key: {task_key}, the field of taskid: {taskid} and the feild of id: " \
+               f"{id} must be with equal value. "
```

Comment:
please use the `self.file_path` in the err msg so the user will know in which PB the error is.